rstudio/rstudio
# reduce brightness of initial desktop window background to match GWT background

### Intent

Addresses [Feature Request: To dull the colour of RStudio Launch Screen or Splash Screen](https://github.com/rstudio/rstudio/issues/13768)

### Approach

During startup of Electron, we first see the background colour passed to `new BrowserWindow` (which was pure white), then we see a grey background gradient shown by GWT, then we see the actual UI which uses the RStudio theme.

Improved (somewhat) by using the same color for BrowserWindow background as the GWT BACKGROUNDGRADIENT.

This isn't quite so bright, and reduces flashing between the first stage and the second. It's not perfect, because GWT actually displays a mild gradient in the background, not a solid colour, but this matches the end (darkest) colour show in that gradient.

This initial background colour also shows through in the startup pages (shown when startup is taking some time):

Old:

<img width="1253" alt="screenshot of startup screen with old solid white background" src="https://github.com/rstudio/rstudio/assets/10569626/c0dca297-36af-40df-aefb-2a7e6db4a7ae">

New:

<img width="1253" alt="screenshot of startup screen with new grey background" src="https://github.com/rstudio/rstudio/assets/10569626/b4962bc4-48e1-4751-8357-5c0cdf7a38ab">

### Automated Tests

None

### QA Notes

Visual check to confirm it's not quite as bright during startup. This is desktop (Electron) only.

### Documentation

None

### Checklist
